src/qml/bitcoin.cpp

```diff
@@ -146,5 +149,7 @@ int QmlGuiMain(int argc, char* argv[])
         return EXIT_FAILURE;
     }
 
+    qInfo() << "Graphics API in use:" << QmlUtil::GraphicsApi(qobject_cast<QQuickWindow*>(engine.rootObjects().first()));
```

Comment:
Since we expecting a window I'd also do:

```cpp
auto window = qobject_cast<QQuickWindow*>(engine.rootObjects().first());
if (!window) return EXIT_FAILURE;

qInfo() << "Graphics API in use:" << QmlUtil::GraphicsApi(window);
```

Reply:
Thanks! [Updated](https://github.com/bitcoin-core/gui-qml/pull/32#issuecomment-908914269).